### DISCUSSION TITLE
Add custom properties to chosen floating windows

### DISCUSSION THREAD

**Discussion by Kirinko** [qtile/qtile#3122] [Ideas]
Hi there! I typically use default options for floating windows. But recently I installed synapse for quick approach to recent files and I want to give this app a unique _border_width_ property. What is the easiest and proper way to do so? It's easy to change options globally for all floating windows. But if I need make a change only for some of them?

**Comment by m-col** [comment]
Hello! I'm afraid that there is no way to specify borders on specific windows like you want. I'm not even sure it's possible with some config hacking unfortunately. As that is the case, it would be good to have the ability to do as as part of the `Floating` layout. If you'd be interested in taking a stab at implementing it, I'd be happy to help out.

**Reply by Kirinko** [reply]
Thank you for a quick answer. Well, I wish I had better skill in python and programming in general. Nowadays, I'm just casual user. So I would appreciate if anyone experienced could suggest some twicks.
